public/app/features/integrated-alerting/components/Alerts/AlertsActions/AlertsActions.styles.ts

```diff
@@ -0,0 +1,69 @@
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+
+export const getStyles = (theme: GrafanaTheme) => {
+  const size = 16;
+
+  const hoverColor = theme.isLight ? theme.palette.gray95 : theme.palette.gray15;
+
+  return {
+    button: css`
+      width: ${size}px;
+      height: ${size}px;
+      background: transparent;
+      border: none;
+      padding: 0;
+      margin: 0;
+      outline: none;
+      box-shadow: none;
+      display: inline-flex;
+      align-items: center;
+      justify-content: center;
+      position: relative;
+      z-index: 0;
+      margin-right: ${theme.spacing.xs};
+
+      &[disabled],
```

Comment:
I tested `&[disabled]` - it doesn't seem to work. Safe to remove, since `emotion` does a pretty good job catering for browser inconsistencies.